flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/data/RowDataUtil.java

```diff
@@ -63,6 +65,27 @@ public static Object convertConstant(Type type, Object value) {
         return ByteBuffers.toByteArray((ByteBuffer) value);
       case BINARY: // byte[]
         return ByteBuffers.toByteArray((ByteBuffer) value);
+      case STRUCT: // struct
```

Comment:
What is this used for? Right now we don't have struct default values so I don't think this will be exercised.